tests/shared/DevelopmentTestDefs.js

```diff
@@ -46,7 +46,7 @@ gpii.tests.development.testDefs = [{
         configName: "gpii.config.development.local",
         configPath: "%gpii-universal/gpii/configs"
     },
-    gradeNames: ["gpii.test.common.testCaseHolder"],
+    //gradeNames: ["gpii.test.testCaseHolder"],
```

Comment:
Stray commented out definition

Reply:
Thanks, that was left over from a fairly serious goof with a broken environment where that grade was required.